### PR TITLE
feat(files): woc_elytras

### PR DESCRIPTION
**Pull request by Chi-a11y**
<description_pr>

By checking the following boxes with an X, you ensure that:

- [X] The pack was tested ingame in at least one device.
- [x] The pack is an existing BT pack, is a missing pack from VT or is an accepted pack/change in a discussion.
- [X] The pack code follows the style guide.
- [X] The commits follow the contribution guidelines.
- [X] The PR follows the contribution guidelines.

- [X] (Optional) Tested in Windows
- [X] (Optional) Tested in Android
- [ ] (Optional) Tested in iOS
- [ ] (Optional) Tested in any console
- [ ] (Optional) Tested in BDS

### CODE REVIEW

**Review by NSPC911 — COMMENTED** [review]
seems fine

**Review by NSPC911 — CHANGES_REQUESTED** [review]
Make sure to add the description key even though it might be empty, so just add a `"description": ""` to each pack

File: resource_packs/packs.json
Comment (SquatchHunter) [inline resource_packs/packs.json:1]: as NSPG pointed out, make sure to include a description tag. And I believe you may need to correct how these are laid out to be in line with how the coloured slimes above (in the pack.json file) are laid out. 
That being each one with its own ID, name & description but DrAv may change how categories such as World of Colour work.

Comment (Chi-a11y) [reply resource_packs/packs.json:1]: i'm not quite sure what should I put since i've checked the VT's and it didn't have any?

Comment (Chi-a11y) [reply resource_packs/packs.json:1]: i've finished it, i'll wait for re-review

**Comment by NSPC911** [comment]
lgtm, need @drav0011 's review to merge anyways

**Review by NSPC911 — APPROVED** [review]
lgtm, need @drav0011 's review to merge anyways

**Review by SquatchHunter — CHANGES_REQUESTED** [review]

**Review by SquatchHunter — APPROVED** [review]

**Review by SquatchHunter — COMMENTED** [review]
ltgm

**Review by drav0011 — APPROVED** [review]
lgtm

**Comment by drav0011** [comment]
Next time do not forget to link the related issue or discussion that you are working on @Chi-a11y
Anyways great work

**Pull request merged**